### PR TITLE
feat(user-page): create and add RepoList component

### PR DESCRIPTION
**Pull request by trybick**
#### Summary

Resolves: [On user page, add more details to repo list](https://github.com/f1v/full-git-commit-history-app/projects/1#card-52574360)

- Remove original d3 chart from user page
- Create component for Repos List and add details for display to it

Screen Shot 2021-01-11 at 11 52 05 AM

#### Checklist

- [x] Branch is in format `TYPE/scope/description`
- [x] PR title is in semantic format `TYPE(scope): description`
- [x] Commits Messages are in semantic format `TYPE(scope): description`
- [x] Has Summary
- [x] Has Labels

### CODE REVIEW

File: src/components/repo-list/RepoList.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        </Link>
+        {starsForDisplay}
+      </Flex>
+
+      <Text fontSize="14px" mb="10px">
+        {description}
+      </Text>
+
+      <Text fontSize="14px">{language}</Text>
+      <Text fontSize="14px">{updatedAtForDisplay}</Text>
```
Comment (davidholyko) [inline src/components/repo-list/RepoList.js:40]: maybe add text "last updated: "

File: src/components/repo-list/RepoList.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+          <Text fontSize="24px">{name}</Text>
+        </Link>
+        {starsForDisplay}
+      </Flex>
+
+      <Text fontSize="14px" mb="10px">
+        {description}
+      </Text>
+
+      <Text fontSize="14px">{language}</Text>
```
Comment (davidholyko) [inline src/components/repo-list/RepoList.js:39]: I think we can expand on this language thing. maybe `javascript` would be black text and rounded yellow background color and so on.

**Review by davidholyko — APPROVED** [review]
LGTM. I have a comment and a minor addition request

File: src/components/repo-list/RepoList.js (hunk 2 above)
Comment (trybick) [reply src/components/repo-list/RepoList.js:39]: Sounds good, let's get this PR in first to avoid merge conflicts. I will make a ticket for this.

**Pull request merged**